src/screens/Bridge/Exchange-CrossChain/OfferCard.js

```diff
@@ -128,10 +129,16 @@ function OfferCard({
 
   const highFee = isHighGasFee(chain, feeInfo?.fee, feeInfo?.gasToken, chainRates, fiatCurrency, gasThresholds);
 
-  const buttonTitle = formatTokenValue(offer.toAmount, offer.toAsset.symbol) ?? '';
-
-  // eslint-disable-next-line i18next/no-literal-string
-  const title = `${buttonTitle}  •  ${formattedFiatValue || ''}`;
+  const title = React.useMemo(() => {
+    if (crossChainTxs) {
+      const crossChainButtonTitle = `${nFormatter(Number(offer.toAmount), 4)} ${offer.toAsset.symbol}` ?? '';
+      // eslint-disable-next-line i18next/no-literal-string
+      return `${crossChainButtonTitle}  •  ${formattedFiatValue || ''}`;
```

Comment:
Can you replace `''` with `` to remove the eslint warning?

Reply:
I can't replace '' to ``.

Reply:
Can you escape using backslashes?

Reply:
Here the issue is "  •  ". it's not considering in string.

other solution is 
```
  const title = `${crossChainButtonTitle} _ ${formattedFiatValue || ''}`;
  return title.replace('_', ' • ');
```
is it good for string?

Reply:
Ahhh okay i didn't realise that was the issue there. I think if this is the case, let's leave it as it is.